js/parts-3d/SVGRenderer.js

```diff
@@ -274,26 +276,34 @@ cuboidMethods = merge(element3dMethods, {
     },
     animate: function (args, duration, complete) {
         if (defined(args.x) && defined(args.y)) {
-            var paths = this.renderer[this.pathType + 'Path'](args);
+            var paths = this.renderer[this.pathType + 'Path'](args), forcedSides = paths.forcedSides;
             this.singleSetterForParts('d', null, paths, 'animate', duration, complete);
             this.attr({
                 zIndex: paths.zIndexes.group
             });
+            // If sides that are forced to render changed, recalculate colors.
+            if (forcedSides !== this.forcedSides) {
+                this.forcedSides = forcedSides;
+                cuboidMethods.fillSetter.call(this, this.fill);
+            }
         }
         else {
             SVGElement.prototype.animate.call(this, args, duration, complete);
         }
         return this;
     },
     fillSetter: function (fill) {
-        this.singleSetterForParts('fill', null, {
+        var elem3d = this;
+        elem3d.forcedSides = elem3d.forcedSides || [];
+        elem3d.singleSetterForParts('fill', null, {
             front: fill,
-            top: color(fill).brighten(0.1).get(),
-            side: color(fill).brighten(-0.1).get()
+            // Do not change color if side was forced to render.
+            top: color(fill).brighten(elem3d.forcedSides.includes('top') ? 0 : 0.1).get(),
```

Comment:
@Izothep Sorry for writing after the merge but it came to me just now. Is the usage of `includes` safe? It's not IE friendly and as far as I have checked oldie-polyfill doesn't cover this.

Reply:
I may change it to classic indexOf